src/Plugin/Queue/Workers/Kafka/KafkaWorker.php

```diff
@@ -208,8 +215,13 @@ private function mapMessages(array $batch): array {
 	private function handleRow(array $message): array {
 		$row = [];
 		foreach ($this->fields as $fieldName => $fieldType) {
-			if (isset($message[$fieldName])) {
-				$row[$fieldName] = $this->morphValuesByFieldType($message[$fieldName], $fieldType);
+			$inputKeyName = $fieldName;
+			if ($this->customMapping !== []
+				&& isset($this->customMapping[$fieldName])) {
```

Comment:
isset should be enough?

Reply:
yes

Reply:
In case isset is enough let's remove `$this->customMapping !== []` cuz it does not do anything, isset is enough